example/exporter/kube-apiserver-servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "monitoring.coreos.com/v1alpha1"
+kind: "ServiceMonitor"
+metadata:
+  name: kube-apiserver
+  namespace: monitoring
+  labels:
+    app: kube-apiserver
+spec:
+  namespaceSelector:
+    matchNames:
+      - default
+  selector:
+    matchLabels:
+      component: apiserver
+      provider: kubernetes
+  endpoints:
+  - scheme: "https"
+    tlsConfig:
+      insecureSkipVerify: true
```

Comment:
if you specify the `caFile` and `serverName` here then there is no need to skip the cert verification here